### ISSUE TITLE
Investigate low performance issues (extreme)

### ISSUE THREAD

**Issue by ghost** [emilyinure/solace-csgo#11]
5 ennemies peeking = no more fps, same when spectating someone.

I literally cannot play, maybe it is how old cs:go performance was back then.

**Comment by emilyinure** [comment]
Yes I really cant stand coding for desync, may be coming out with a legacy p2c soon with some friends though, sorry if this repo dies
